demo/index.html

```diff
@@ -10,13 +10,14 @@
     <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
 
     <title>mirador-viewer demo</title>
-
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>

```

Comment:
I don't think that is needed. I'm pretty sure I that I included it for testing purposes, and forgot to get rid of it. I just tested the demo without it installed, and everything worked. I will update the PR momentarily.